kibana-reports/server/routes/visualReport.ts

```diff
@@ -50,13 +50,18 @@ export default function (router: IRouter) {
       try {
         let report = request.body;
         const reportParams = report.report_params;
-
+        // placeholder since header & footer are not currently in data schema
+        const testHeader = 'Test report header sample text';
+        const testFooter = 'Test report footer sample text';
```

Comment:
Can you mark it as TODO in the comments.

Reply:
Sure, good call